src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -70,17 +76,22 @@ const Form = React.memo(({ setSuccess, success }) => {
     const handleSubmit = async e => {
         e.preventDefault();
         setCanSubmit(false);
-        setSuccess(true);
-        const obj = {
+        const data = {
             first_name: firstName,
             last_name: lastName,
             email,
             institution_name: institutionName,
             institution_type: institutionType,
             agree_to_email: agreeToEmail,
         };
-
+        const response = await submitAcademiaForm(data);
         //TODO: communicate with the backend here
+        if (response.success) {
```

Comment:
Does that TODO need to be updated?

Reply:
What about `setSuccess(response.success)`? :)

Reply:
We still need to also enable the button on a resubmit which would lead to:

```
setCanSubmit(!response.success)
setSuccess!response.success)
```